tracing-jersey/build.gradle

```diff
@@ -17,17 +17,40 @@
 apply plugin: 'com.palantir.external-publish-jar'
 apply plugin: 'com.palantir.revapi'
 
+versionsLock {
+    disableJavaPluginDefaults()
+}
+
 dependencies {
-    api "org.glassfish.jersey.core:jersey-server"
+    implementation "org.glassfish.jersey.core:jersey-server", {
+        version {
+            strictly '2.22.2'
+        }
+    }
     api project(":tracing")
 
     implementation 'com.google.guava:guava'
-    implementation 'jakarta.ws.rs:jakarta.ws.rs-api'
+    implementation 'javax.ws.rs:javax.ws.rs-api', {
+        version {
+            strictly '2.1.1'
+        }
+    }
     implementation project(':tracing-api')
 
-    testImplementation "io.dropwizard:dropwizard-testing"
+    testImplementation "io.dropwizard:dropwizard-testing", {
+        version {
+            strictly '2.0.10'
+        }
+    }
```

Comment:
can we comment why these versions are strictly pinned?

Reply:
yup